Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -998,10 +996,8 @@ int RenderLayerScrollableArea::horizontalScrollbarHeight(OverlayScrollbarSizeRel
     if (m_hBar && m_hBar->isOverlayScrollbar() && (relevancy == IgnoreOverlayScrollbarSize || !m_hBar->shouldParticipateInHitTesting()))
         return 0;
 
-    if (!m_hBar && m_layer.renderBox() && !(m_layer.renderer().style().scrollbarGutter().isAuto
-        || ScrollbarTheme::theme().usesOverlayScrollbars()) && !isHorizontalWritingMode) {
+    if (!m_hBar && !(scrollbarGutterStyle().isAuto || ScrollbarTheme::theme().usesOverlayScrollbars()) && !isHorizontalWritingMode)
```

Comment:
It’s slightly better for performance to put tests likely to fail first when it’s a series separated by `&&` and the computations are not trivial. I am thinking `!isHorizontalWritingMode` should be earlier to save at least a virtual function call.

Reply:
Will make that change now